doc/howto/network_ipam.md

```diff
@@ -0,0 +1,48 @@
+(network-ipam)=
+# How to display IPAM information of a LXD deployment
+
+{abbr}`IPAM (IP address management)` is a method used to plan, track, and manage the information associated with a computer network's IP address space. In essence, it's a way of organizing, monitoring, and manipulating the IP space in a network.
+
+You can display {abbr}`IPAM` information like so:
+
+```bash
+lxc network list-allocations
+
+Flags:
+    --all-projects          Run against all projects
+    -p, --project string    Run again a specific project (default "default")
+
+```
```

Comment:
Usually we don't just list the flags (there's the help command for that) but describe the relevant ones in text.
So this could be something like:

"By default, this command shows the IPAM information for the `default` project.
You can select a different project with the `--project` flag, or specify `--all-projects` to display the information for all projects."